### ISSUE TITLE
"Code after RETURN statement cannot be executed" false positive when used with Switch

### ISSUE THREAD

**Issue by dpegasusm** [WordPress-Coding-Standards/WordPress-Coding-Standards#963]
When a return exists in a switch statement the error: 'Code after RETURN statement cannot be executed' is thrown, despite there being multiple pathways for the script to take.

**Comment by JDGrimes** [comment]
That error comes from the sniff `Squiz.PHP.NonExecutableCode.Unreachable`, which is part of PHPCS itself and not a part of WPCS. So it may actually be an upstream issue. Can you post a minimal example demonstrating the problem? Do you still get the error when running just that sniff over the file? (`phpcs --standard=Squiz --sniffs=Squiz.PHP.NonExecutableCode`) If so, that's an indicator that it isn't related to WPCS specifically.

**Comment by jrfnl** [comment]
What @JDGrimes said.

I've most often come across this issue with code along the lines of the below:
```php
switch( $something ) {
	case 'A':
		return 'A';
		break; // <- This is the nonExecutable & unreachable code the sniff refers to.
	case 'B':
		// ... etc
}

```

**Comment by dpegasusm** [comment]
@jrfnl that is exactly what i am seeing as well. Thanks for the example. It seems like it's an upstream issue @JDGrimes I will open an issue there.

**Comment by jrfnl** [comment]
@dpegasusm In that case, it's actually not an issue or a false positive. The `break` is unreachable and therefore correctly being reported.
You can safely remove the `break` as the `return` breaks out of the switch already.
